### ISSUE TITLE
Streams

### ISSUE THREAD

**Issue by St4B** [android10/Android-CleanArchitecture-Kotlin#13]
I am not in a so good level in using kotlin, so maybe i am missing something,  but are not we losing some capabilities by not using streams of data?

**Comment by mateherber** [comment]
I think the message here is not "do not use rxJava" but "you may not use rxJava solely as a Promise/Future library or for handling threads" if you have use cases which resonates well with data streams I think it is still wise to use rx.

**Comment by Zhuinden** [comment]
Yeah, zipping event streams or hiding listeners as Observable (allowing filters and etc) is still good Rx use case
